### ISSUE TITLE
Emulator 4.2.1 doesn't show transcript file generated with chatdown

### ISSUE THREAD

**Issue by KrisvanderMast** [Microsoft/BotFramework-Emulator#1256]
## Tool
Name: [Chatdown]
Version: *1.0.10*
OS: *Windows 10*

## Describe the bug
A couple of months ago transcripts didn't load when I used chatdown latest. Reverting to version 1.0.10 did the trick as described in https://github.com/Microsoft/botbuilder-tools/issues/702. However today I tried (with both versions of chatdown) to generate a transcript file but it doesn't open in the bot emulator (version 4.2.1). 

The bot emulator gives a notification stating the following "Error creating a new conversation in transcript mode: Error: Error while feeding transcript on disk to conversation: Unexpected token ? in JSON at position 0

## To Reproduce
1. use chatdown greeting.chat > greeting.transcript
2. Open the bot emulator
3. File > open transcript....
4. Select the transcript file
5. There's no chat conversation being shown

## Expected behavior
I would expect to see the chat messages between bot and user. It used to work.

## Additional context

[bug]

**Comment by tonyanziano** [comment]
@KrisvanderMast we encountered a similar issue a while back, and it was due to the fact that VSCode's integrated terminal uses a different encoding -- UTF-16le I believe -- when generating the `.transcript` file. It was the initial [Byte Order Mark](https://en.wikipedia.org/wiki/Byte_order_mark) at the beginning of the file that was causing the issue. We did end up using a library to handle the BOM, and so it seems like this might be a regression.

I'll investigate.
